addons/rating/models/rating_mixin.py

```diff
@@ -76,10 +76,10 @@ def write(self, values):
             result = super(RatingMixin, self).write(values)
             for record in self:
                 if record._rec_name in values:  # set the res_name of ratings to be recomputed
-                    res_name_field = self.env['rating.rating']._fields['res_name']
+                    res_name_field = self.env['rating.rating'].sudo()._fields['res_name']
```

Comment:
I'm not sure this one is usefull, as we try to get property of a python class. Check access are not applied here.